manager/ui/war/tomcat8/src/main/webapp/WEB-INF/keycloak.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
And this too. I think it would be better if we could have keycloak.json outside of the WAR zip.

Reply:
Regarding external `keycloak.json`: 

This doc seems to be for the OSGi adaptor, but I guess it should work for all of them; might require a bit of digging, but looks doable -- https://www.keycloak.org/docs/latest/securing_apps/#config_external_adapter